### PR TITLE
moved rubocop to do to rubocop.yml

### PR DESCRIPTION
**Pull request by rajastakilos**
Per Sungwon's instructions.

### CODE REVIEW

**Review by dtrypt — APPROVED** [review]

File: .rubocop.yml [hunk 1]
```diff
@@ -22,4 +22,63 @@ Metrics/BlockLength:
   Exclude:
     - 'Rakefile'
     - '**/*.rake'
-    - 'spec/**/*.rb'
\ No newline at end of file
+    - 'spec/**/*.rb'
+
+#Note: Beginning of old .rubocop_todo.yml below this line.
+
+# Offense count: 1
```
Comment (sungwon) [inline .rubocop.yml]: Take out these offense count comments since they are no longer relevant.

File: .rubocop.yml [hunk 2]
```diff
@@ -22,4 +22,63 @@ Metrics/BlockLength:
...
+
+# Offense count: 1
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: nested, compact
+Style/ClassAndModuleChildren:
+  Exclude:
+    - 'app/controllers/users/omniauth_callbacks_controller.rb'
+
+# Offense count: 13
+Style/Documentation:
```
Comment (sungwon) [inline .rubocop.yml:51]: Instead of excluding everything (because we will have to do it for future files as well), let's disable this check.

**Review by sungwon — CHANGES_REQUESTED** [review]
See the 2 in-line comments. Also, after making changes, push to the same branch on origin. This will update the pull request so you don't have to make another one.

File: .rubocop.yml (hunk 2 above)
Comment (rajastakilos) [inline .rubocop.yml:51]: Hi sungwon, 

Do you mean just for Style/Documentation?

Comment (sungwon) [reply .rubocop.yml:51]: Yes

File: .rubocop.yml [hunk 3]
```diff
@@ -22,4 +22,56 @@ Metrics/BlockLength:
...
+  Max: 15
+
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: nested, compact
+Style/ClassAndModuleChildren:
+  Exclude:
+    - 'app/controllers/users/omniauth_callbacks_controller.rb'
+
+Style/Documentation:
+  Enabled: false
```
Comment (sungwon) [inline .rubocop.yml:52]: Now that enabled is false, we don't need the Exclude list of files below it.

**Review by sungwon — CHANGES_REQUESTED** [review]

File: .rubocop.yml (hunk 3 above)
Comment (rajastakilos) [reply .rubocop.yml:52]: Good call. Will remove

**Review by sungwon — APPROVED** [review]

**Pull request merged**